docs/streams/stream-examples/realtime-updates.md

```diff
@@ -74,8 +105,172 @@ async function createCollection () {
     await console.log("Collection creation did not succeed due to " + e);
   }
 }
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+
+```py
+if __name__ == '__main__':
+
+    # Step1: Open connection to GDN. You will be routed to closest region.
+    print(f"\n1. CONNECT: federation: {URL}")
+    client = C8Client(protocol='https', host=URL, port=443, apikey=API_KEY, geofabric=GEO_FABRIC)
+
+    # Step 2: Create a collection if one does not exist
+    print(f"\n2. CREATE_COLLECTION: region: {URL},  collection: {COLLECTION_NAME}")
+    if client.has_collection(COLLECTION_NAME):
+        collection = client.collection(COLLECTION_NAME)
+    else:
+        collection = client.create_collection(COLLECTION_NAME, stream=True)
+
+    # Subscribe to be notified when changes are made to collection.
+
+    def create_callback():
+        def callback_fn(event):
+            pp.pprint(event)
+            return
+
+        client.on_change(COLLECTION_NAME, callback=callback_fn, timeout=15)
+```
+
+</TabItem>
+</Tabs>
+
+
+### Step 4. Add Data and Subscribe to Collection
+
+Add to the collection the information we defined in the `data` variable, then subscribe to the collection.
+
+In this example, we are adding IP addresses to blacklist.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
 async function insertData () {
-  console.log(`\n3. Insert data in region ${globalUrl}`);
+  console.log(`\n3. Insert data`);
+  await client.insertDocumentMany(collectionName, data);
+}
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+
+```py
+if __name__ == '__main__':
+
+    # Step3: Subscribe to receive documents in realtime (PUSH model)
+    print(f"\n3. SUBSCRIBE_COLLECTION: region: {URL},  collection: {COLLECTION_NAME}")
+    rt_thread = threading.Thread(target=create_callback)
+    rt_thread.start()
+    time.sleep(10)
+    print(f"Callback registered for collection: {COLLECTION_NAME}")
+
+    # Step4: Subscribe to receive documents in real-time (PUSH model)
+    print(f"\n4. INSERT_DOCUMENTS: region: {URL},  collection: {COLLECTION_NAME}")
+    client.insert_document(COLLECTION_NAME, document=data)
+
+    # Step5: Wait to close the callback.
+    print("\n5. Waiting to close callback")
+    rt_thread.join(2)
+```
+
+</TabItem>
+</Tabs>
+
+
+### Step 5. Delete Data
+
+Delete the data from the collection. If the collection becomes empty, the collection is deleted too.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
+async function deleteData () {
+  console.log("\n4. Delete data");
+  await client.deleteCollection(collectionName);
+}
+(async function () {
```

Comment:
This is the part where the function actually runs. I strongly suspect based on other examples I've worked on (double check with @koshy-thomas or test it yourself) that we need to have the run commands in this function up with each of the steps.

As it is, the user gets no feedback when doing the JavaScript step by step.